site/en/blog/permissions-chip/index.md

```diff
@@ -68,16 +65,17 @@ how Windows users reacted to the geolocation and the notification prompts on sit
 Given an ignore/dismiss rate of approximately 85%, and especially given how much the prompt stands out and
 insists on users making a decision immediately, there is a conflict between the level of urgency
 assumed by the browser versus the user's preference for waiting to make a decision. This creates the
-perception that it's "annoying" for a site to ask for a permission.
+perception that it's "annoying" for a site to ask for a permission, apart from potential extra
+things users need to react to like cookie consent banners, newsletter signups, etc.
 
 ## New design
 
-From Chrome 97 on, we therefore introduced an animated chip UI which appears next to the lock
+From Chrome 98 on, we therefore introduced an animated chip UI which appears next to the lock
 whenever a permission is requested. This consists of an icon and label describing the permission being
 requested. Our aim was to improve the experience of web browsing while avoiding permission requests
 that are generally unnecessary for the vast majority of users and frequently ignored or dismissed.
 
-{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/SsFtKKnZzbkAovNkX4Lt.png", alt="Google search engine results page showing a search for pizza and the permissions chip with 'Use your location?' label.", width="653", height="229" %}
+{% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/4rH9koYThnPvEgLu4mF6.mp4", autoplay=true, muted=true, playsinline=true, loop=true%}
```

Comment:
If the alt property was wrong, please replace it rather than removing it.

Reply:
This replaces an image with a video. Videos don't have `alt`.